ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/duplicate-vrfs-id-conflict.yml

```diff
@@ -44,8 +44,4 @@ tenants:
             ip_address_virtual: 10.0.100.1/24
 
 expected_error_message: >-
-  Found duplicate objects with conflicting data while generating configuration for BGP VRFs defined under network services.
-  {'name': 'VRF1', 'rd': '192.168.255.101:1', 'route_targets': {'import': [{'address_family': 'evpn', 'route_targets': ['1:1']}],
-  'export': [{'address_family': 'evpn', 'route_targets': ['1:1']}]}} conflicts with {'name': 'VRF1', 'rd': '192.168.255.101:2',
-  'route_targets': {'import': [{'address_family': 'evpn', 'route_targets': ['2:2']}], 'export': [{'address_family': 'evpn',
-  'route_targets': ['2:2']}]}}.
+  Found duplicate objects with conflicting data while generating configuration for Vrfs. {'name': 'VRF1', 'vni': 1} conflicts with {'name': 'VRF1', 'vni': 2}.
```

Comment:
This happens because the order of rendering changes, so we catch the duplicate under VXLAN VRFs instead of under BGP VRFs.

Reply:
try to make vrf_vni unique and keep vrf_id the same.